### DISCUSSION TITLE
Message curieux

### DISCUSSION THREAD

**Discussion by SebastienMeunier** [thelfer/tfel#398] [Q&A]
Hello,

After having, I believe successfully, installed MFront, I have the following error message that I do not understand:
`./tfel-TFEL-4.1.0/install/bin/mfront --help
./tfel-TFEL-4.1.0/install/bin/mfront: error while loading shared libraries: libTFELMFront.so.4.1.0: cannot open shared object file: No such file or directory`
Could you explain me why please ?

**Comment by thelfer** [comment]
Hi @SebastienMeunier. Did you set your LD_LIBRARY_PATH environment ?

**Reply by SebastienMeunier** [reply]
I tried to inspire from the Elodie Bosso procedure (https://github.com/thelfer/tfel/blob/master/INSTALL-EDF-calibre) and I have added  a profile_mfront.sh containing:
export TFELHOME=${HOME}'/opt/mfront/tfel-TFEL-4.1.0/install/'

export PATH=$TFELHOME/bin:$PATH
export LD_LIBRARY_PATH=$TFELHOME/lib:$LD_LIBRARY_PATH

But apparently, this is not enough

**Reply by thelfer** [reply]
What is the output of the following commands:

```
$ which mfront
$ echo $LD_LIBRARY_PATH
```

**Reply by SebastienMeunier** [reply]
Finally, I succeeded by typing manually:

export TFELHOME=${HOME}"/opt/mfront/tfel-TFEL-4.1.0/install/"
export PATH=$TFELHOME/bin:$PATH
export LD_LIBRARY_PATH=$TFELHOME/lib

But then, I tried to launch mtest files generated by failure of code_aster computations and then, error messages :
mtest-4.1.0 ${HOME}/mtest/gtn-1.mtest
terminate called after throwing an instance of 'std::runtime_error'
  what():  SingleStructureSchemeParser::SingleStructureScheme : error while parsing file '/home/F82953/mtest/gtn-1.mtest' at line '4'.
ExternalLibraryManager::getGenericBehaviourFunction: could not load generic behaviour function 'astergtn_PlaneStrain' (src/libAsterBehaviour.so: undefined symbol: astergtn_PlaneStrain)
Abandon

**Reply by thelfer** [reply]
Looks like there is an interface mismatch.

It 's because the new versions of aster relies on the `generic` interface. The `aster` interface is doomed to be deprecated.
